src/components/decks/Deck.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    class="border border-gray bg-white mt-4 mb-4">
+    <div class="p-2 text-xs">
+      <div class="m-0 font-bold text-xl">
+        <router-link :to="linkTarget" class="text-black">{{ deck.title }}</router-link>
+        <span class="float-right">
+          <deck-dice :dice="deck.dice" />
+        </span>
+      </div>
+      <div class="clear-right">
+        <span class="text-sm">
+          {{ deck.user.username }}
```

Comment:
Please create a component to output the username that concatenates the username with a lighter gray badge (like "**Skaak**#1234"). Eventually, I'll be swapping this out for a link, so having a component will make life easier for that.

Reply:
Done, hope it looks fine. If temporary anyway that's probably ok:
![image](https://user-images.githubusercontent.com/762494/103447670-149b9000-4c86-11eb-9b3b-82756a73bd0d.png)